### ISSUE TITLE
Refresh should restore in-progress game via localStorage

### ISSUE THREAD

**Issue by thejoshwolfe** [dgoffredo/kasuba#1]
mobile browsers aggressively trash the state of open tabs and effectively refresh the page on you without warning. It would be nice if it resumed the game where you had it last.

https://wolfesoftware.com/loops/ does this.

**Comment by dgoffredo** [comment]
Good idea, thanks for the suggestion. I've been working on this on a local branch. The trick is small modifications to cube.js that will allow the specification of non-preset, non-blank cells.  You're right that local storage is appropriate, as IndexedDb would be overkill.
